### ISSUE TITLE
cmake toolchain misaliment 

### ISSUE THREAD

**Issue by ivand58** [espressif/esp32c3-direct-boot-example#6]
Hi 
There is a problem, probably with the objcopy location:
```
[ 85%] Linking C executable hello_world
[ 85%] Built target hello_world
   text	   data	    bss	    dec	    hex	filename
   5092	    136	    160	   5388	   150c	hello_world
[ 85%] Built target hello_world-size
[100%] Generating hello_world.bin
copy from `hello_world' [elf32-little] to `/home/i.../hello_world.bin' [binary]
/usr/bin/objcopy: Unable to recognise the format of the input file `hello_world'
```

**Comment by ivand58** [comment]
after manual fix of the cashed cmake, the size seems to be incorrect:
```
          24292 hello_world*
1070071812 hello_world.bin*
```

**Comment by ivand58** [comment]
it is fixed in branch #5
